app/services/AssessmentService.scala

```diff
@@ -34,84 +35,24 @@ class AssessmentService @Inject()(
                                    val authorisationManagementApi: AuthorisationManagementApi,
                                    val mdtpDashboardManagementApi: MdtpDashboardManagementApi,
                                    @Named("authedAssessmentEndpointEnabled") val authedAssessmentEndpointEnabled: Boolean
-                                 )(implicit executionContext: ExecutionContext){
+                                 )(implicit executionContext: ExecutionContext) extends Cats {
```

Comment:
what are we getting from Cats here?

Reply:
removing all the implicit parameters at the method level.

Reply:
it's getting you instances for Monad[Future] so we can remove this stuff from all over the place `F: cats.Functor[scala.concurrent.Future], f: cats.Monad[scala.concurrent.Future]` I happen to know because I also did the same on my own branch :-D

Reply:
nice one 👍